sweep/tx_input_set.go

```diff
@@ -201,8 +201,8 @@ func (t *txInputSet) Budget() btcutil.Amount {
 // DeadlineHeight gives the block height that this set must be confirmed by.
 //
 // NOTE: this field is only used for `BudgetInputSet`.
-func (t *txInputSet) DeadlineHeight() fn.Option[int32] {
-	return fn.None[int32]()
+func (t *txInputSet) DeadlineHeight() int32 {
+	return 0
```

Comment:
See the above comment. I also think `InputSet` is a redundant interface and instead be put inside the aggregator, then we can avoid the validations below, since it only cares about the budget and deadline, and they can be derived from the set of inputs directly. Future refactoring tho.

Reply:
Right but we can't ever confirm anything by height 0, so are we treating 0 as if it's `None`? If so, I think this is a step in the wrong direction for what we want to do.

Reply:
Can we add a comment here that says it's being deprecated, then?